src/computeSourceMap.ts

```diff
@@ -0,0 +1,22 @@
+import sourceMap, {RawSourceMap} from "source-map";
+
+/**
+ * Generate a simple source map indicating that each line maps directly to the original line.
+ */
+export default function computeSourceMap(code: string, filePath: string): RawSourceMap {
+  const mapGenerator = new sourceMap.SourceMapGenerator({file: filePath});
+  let numLines = 1;
+  for (let i = 0; i < code.length; i++) {
+    if (code[i] === "\n") {
+      numLines++;
+    }
+  }
+  for (let line = 1; line <= numLines; line++) {
```

Comment:
You only need to call `addMapping` for changed lines, I think.

Reply:
I tried debugging in WebStorm with no mappings at all and it didn't work, so maybe in at least some cases you need at least one line per mapping? Certainly in some configurations/situations, Sucrase won't change the code at all, and I still want the resulting file to be debuggable.

Also, at least for this simple approach (which may or may not be reasonable in the long run), it's a lot easier to just do one mapping per line rather than checking if the line changed.

Reply:
You may need a mapping for the `{line: 0, column: 0}` for my suggestion to work. Otherwise, the debugger will assume the code before the first mapping is untraceable. Can you try that?

For the interim solution this PR provides, it's nothing to worry about. But it's worth avoiding (if possible) when you get around to implementing column mappings.

Reply:
I just played around with it a bit, and it looks to me like I actually do need to map every line, at least for it to work in WebStorm. With an untransformed code snippet that's 4 lines long, if I just map the first and third line and set breakpoints on every line, it only stops on the lines I explicitly mapped, and grays out the breakpoints on unmapped lines to indicate that they won't do anything:
<img width="244" alt="screen shot 2018-06-17 at 12 59 38 pm" src="https://user-images.githubusercontent.com/211605/41511579-7abeedbc-722e-11e8-8b21-1accf0bf9f8a.png">

This might just be a bug in WebStorm, but obviously I want to do whatever makes tools work. OTOH, this strategy of mapping just the start of each line seems to get everything I want in WebStorm, even variable hovering on unmapped lines (which presumably just looks at the identifier name under the mouse cursor and tries to resolve it).

So for now, I'll leave this as-is.

Reply:
Strange. You might want to notify the WebStorm team. Have you tested that same example in Chrome, too?

Reply:
Just tested it in Chrome now (chrome devtools for node, not running JS on a webpage), and looks like Chrome has the same behavior:

<img width="258" alt="screen shot 2018-06-17 at 8 25 10 pm" src="https://user-images.githubusercontent.com/211605/41517015-f4e0caa0-726c-11e8-80c0-5749414fd33d.png">

The grayed out lines means I can't set a breakpoint, and stepping line-by-line skips those lines. So looks like tools expect at least one mapping per line? Should be pretty easy to include on my end, I think.